### PR TITLE
Fix some deprecation warnings

### PR DESCRIPTION
**Pull request by rcaridade145**
Based on

https://gitlab.gnome.org/GNOME/metacity/commit/5aecb8336c6c90a9ad1bb46295ca60624f5a44aa
https://gitlab.gnome.org/GNOME/metacity/commit/1dcde194923c2f6791de0c6b5d6bd6c3b66b80da
https://gitlab.gnome.org/GNOME/metacity/commit/359f60a6aee5f1338a5edd9f4d84557126c6d46e

### CODE REVIEW

**Comment by rcaridade145** [comment]
That work was based on metacity and a particular commit. I can try and restore that functionality as suggested here https://developer.gnome.org/gtk3/stable/GtkImageMenuItem.html

**Comment by raveit65** [comment]
No, please do not Icons from menus or buttons.
We are MATE and not Gnome......

**Comment by raveit65** [comment]
Why not using 3 commits with meaningful names like gnome does?
1. menu: replacing GtkStock deprecation
2. frames: don't bother painting the background

    GDK (and also GTK+!) do this for us already.
3. theme: libmetacity: avoid deprecated g_memmove

**Comment by rcaridade145** [comment]
I've been working on this and got to the problem discussed here [https://stackoverflow.com/questions/48452717/how-to-replace-the-deprecated-gtk3-gtkimagemenuitem](url) 
left_align_small

This being said i reverted to the old behavior .

**Comment by rcaridade145** [comment]
> Why not using 3 commits with meaningful names like gnome does?
> 
>     1. menu: replacing GtkStock deprecation
> 
>     2. frames: don't bother painting the background
>        GDK (and also GTK+!) do this for us already.
> 
>     3. theme: libmetacity: avoid deprecated g_memmove

Sorry will do that on the next PR's i've got on the pipeline.

File: src/ui/menu.c [hunk 1]
```diff
@@ -60,6 +60,7 @@ struct _MenuData {
 
 static void activate_cb(GtkWidget* menuitem, gpointer data);
 
+
```
Comment (vkareh) [inline src/ui/menu.c]: This change belongs to a commit that's no longer part of this. Please remove

File: src/ui/menu.c [hunk 2]
```diff
@@ -311,6 +312,7 @@ static GtkWidget* menu_item_new(MenuItem* menuitem, int workspace_id)
 	return mi;
 }
 
+
```
Comment (vkareh) [inline src/ui/menu.c]: Same here...

**Comment by raveit65** [comment]
Thanks, for updating PR in single commits.
I just tested it and it runs fine.

**Pull request merged**